app/controllers/sessions_controller.rb

```diff
@@ -1,8 +1,13 @@
 class SessionsController < ApplicationController
+  cattr_accessor :code
   def create
+    SessionsController.token_code(params[:code])
     party = Party.find_by(identifier: request.env["omniauth.params"]["url"]) if request.env["omniauth.params"]["url"]
     if user = User.find_by(spotify_id: spotify_params(request.env['omniauth.auth'])[:spotify_id])
```

Comment:
Can we change spotify_params to not require an argument like we had discussed during our project check in with Josh yesterday?

Reply:
Yeah.Just changed that